docs/4.3/enterprise/ssh_sso.md

```diff
@@ -119,36 +125,6 @@ spec:
       '*': '*'
 ```
 
-## Working with External Email Identity
-
-Along with sending groups, an SSO provider will also provide a user's email address.
-In many organizations, the username that a person uses to log into a system is the
-same as the first part of their email address - the 'local' part. For example, `dave.smith@acme.com`
- might log in with the username `dave.smith`. Teleport 4.2.6+ adds an easy way to
- extract the first part of an email address so it can be used as a username - this
- is the `{% raw %}{{email.local}}{% endraw %}` function.
-
-If the email claim from the identity provider (which can be accessed via `{% raw %}{{external.email}}{% endraw %}`)
-is sent and contains an email address, you can extract the 'local' part of the email
-address before the @ sign like this: `{% raw %}{{email.local(external.email)}}{% endraw %}`
-
-Here's how this looks in a Teleport role:
-
-```yaml
-kind: role
-version: v3
-metadata:
-  name: sso_user
-spec:
-  allow:
-    logins:
-    # Extracts the local part of dave.smith@acme.com, so the login will
-    # now support dave.smith.
-    - '{% raw %}{{email.local(external.email)}}{% endraw %}'
-    node_labels:
-      '*': '*'
-```
-
```

Comment:
Has this section been moved to elsewhere in the docs?

Reply:
Good catch, this is now in https://github.com/gravitational/teleport/pull/3858